src/test/java/features/Dropdown Tests.feature

```diff
@@ -15,7 +15,12 @@ Feature: Dropdown Tests
 	
   @22
   Scenario: 22 Prime NG Dropdown
-    Given I am on the Prime NG Dropdown Page
+    Given I am on the Prime NG Home Page
+    When I click the Components Button
+    Then I am redirected to the Prime NG Setup Page
+      And I verify the Get Started Header exists
```

Comment:
We do not need this here. The framework automatically waits for the page to load, and we aren't trying to test this. It's an extraneous test.